package.json

```diff
@@ -23,7 +23,7 @@
     "preinstall": "node ./scripts/ensure-yarn.js",
     "postinstall": "preconstruct dev && manypkg check",
     "changeset": "changeset",
-    "version-packages": "changeset version && yarn --no-immutable",
+    "version-packages": "changeset version && yarn --mode=\"update-lockfile\"",
```

Comment:
It has been suggested as the faster/better option for this use case (which still isn't 100% safe as it could potentially change unrelated sections of the lockfile but there is no solution for this problem)